training/controllers/function_decorators.py

```diff
@@ -1,13 +1,25 @@
 from functools import wraps
 from flask import session, render_template, request
 
+import os
+from itsdangerous import URLSafeTimedSerializer, BadSignature
+serializer = URLSafeTimedSerializer(os.environ.get('SECRET_KEY'))
```

Comment:
I think you have this already defined in another file

Reply:
Yes, thanks, changed!